packages/maskbook/src/plugins/ITO/assets/formatDate.ts

```diff
@@ -14,7 +14,7 @@ export function datetimeISOString(value: Date): string {
     if (value && value instanceof Date && !isNaN(value.valueOf())) {
         const pad = (value: number) => value.toString(10).padStart(2, '0')
         const date = [value.getFullYear(), value.getMonth() + 1, value.getDate()]
-        const time = [value.getHours(), value.getMinutes(), value.getSeconds()]
+        const time = [value.getHours(), value.getMinutes()]
         return `${date.map(pad).join('-')}T${time.map(pad).join(':')}`
```

Comment:
why not use `import format from 'date-fns/format'`?

Reply:
???

Reply:
Code function meets demand

Reply:
`datetimeISOString()` is useless, just remove it.